src/app/home-page/homepage/homepage.component.ts

```diff
@@ -9,13 +10,14 @@ import { Router } from '@angular/router';
 })
 export class HomepageComponent implements OnInit {
 
-  constructor(private router: Router) { }
+  constructor(private validationService: ValidationService ,private router: Router) { }
```

Comment:
![image](https://user-images.githubusercontent.com/43812715/84049597-ee8e6580-a9ac-11ea-91c6-19f5d7fc8f3e.png)

Reply:
ValidationService, private         not         ValidationService ,private

Reply:
Its private validationService: ValidationService.
The comma is for the next service....

Reply:
My point is you got the space in the wrong place